ccmlib/dse_node.py

```diff
@@ -180,7 +178,10 @@ def start(self,
                 node.watch_log_for_alive(self, from_mark=mark)
 
         if wait_for_binary_proto:
-            self.wait_for_binary_interface()
+            self.watch_log_for("Starting listening for CQL clients", from_mark=self.mark)
```

Comment:
Are we moving away from `wait_for_binary_interface` so we can use the mark? I'd rather add a `mark` parameter to `wait_for_binary_interface` than duplicate the `"Starting listening for CQL clients"`. Alternatively we could define that string as a shared constant somewhere.